### PR TITLE
Install.md documentation for Windows : Add Runtime information for releases

### PR DESCRIPTION
**Pull request by OlivierWienholtz**
Add information regarding Intel runtime for Windows.

Releases & Native builds

### CODE REVIEW

File: INSTALL.md [hunk 1]
```diff
@@ -18,35 +18,61 @@ Set the following environment variables:
...
 ### Environment variables settings under Windows cygwin shell
 
+* If OpenRadioss was build from Source code, load the Intel OneAPI variables prior to launch cygwin. This will load the appropriate 
+  Runtime libraries.
+
+        call "C:\Program Files (x86)\Intel\oneAPI\setvars.bat" intel64 vs2019
+        chdir C:\cygwin64\bin
+        bash --login -i
+
+* If OpenRadioss was doanloaded from GitHub releases,
```
Comment (sebastienVilleneuve) [inline INSTALL.md:56]: here some typo to correct

File: INSTALL.md [hunk 2]
```diff
@@ -18,35 +18,61 @@ Set the following environment variables:
...
 
+* If OpenRadioss was build from Source code, load the Intel OneAPI variables prior to launch cygwin. This will load the appropriate 
+  Runtime libraries.
+
+        call "C:\Program Files (x86)\Intel\oneAPI\setvars.bat" intel64 vs2019
+        chdir C:\cygwin64\bin
+        bash --login -i
+
+* If OpenRadioss was doanloaded from GitHub releases, 
+  Add rhe runtime libraries to PATH in Cygwin.
```
Comment (sebastienVilleneuve) [inline INSTALL.md:57]: here an other type , i guess

**Review by sebastienVilleneuve — CHANGES_REQUESTED** [review]
Hi @OlivierWienholtz 
Some type to correct 

Thanks,
Sébastien

File: INSTALL.md (hunk 1 above)
Comment (OlivierWienholtz) [reply INSTALL.md]: Fixed, thank you for pointing me out.

File: INSTALL.md (hunk 2 above)
Comment (OlivierWienholtz) [reply INSTALL.md]: Fixed as well

**Pull request merged**